lib/tbot/testhelpers/srv.go

```diff
@@ -167,16 +175,23 @@ func MakeBot(t *testing.T, client auth.ClientI, name string, roles ...string) *p
 // - Uses a memory storage destination
 // - Does not verify Proxy WebAPI certificates
 func DefaultBotConfig(
-	t *testing.T, fc *config.FileConfig, botParams *proto.CreateBotResponse, outputs []botconfig.Output,
+	t *testing.T, fc *config.FileConfig, botParams *proto.CreateBotResponse, outputs []botconfig.Output, opts DefaultBotConfigOpts,
 ) *botconfig.BotConfig {
 	t.Helper()
 
 	authCfg := servicecfg.MakeDefaultConfig()
 	err := config.ApplyFileConfig(fc, authCfg)
 	require.NoError(t, err)
 
+	var authServer = ""
+	if opts.UseAuthServer {
+		authServer = authCfg.AuthServerAddresses()[0].String()
+	} else {
```

Comment:
It's more concise to write this with the "else" value as the initial value of the variable.
```go
authServer := authCfg.Proxy.WebAddr.String()
if opts.UseAuthServer {
    authServer = authCfg.AuthServerAddresses()[0].String()
}
```

This should resolve the linter complaint as well.